manifests/manifest-dev-api.yml

```diff
@@ -15,6 +15,7 @@ applications:
       - fecfile-api-redis
       - fecfile-api-creds-dev
     env:
+      USERNAME_PASSWORD_LOGIN_AVAILABLE: False
```

Comment:
@mjtravers  @Elaine-Krauss-TCG  To facilitate testing for shelly, we'll set the environment variable to true on cloud.gov for dev until it's no longer needed.